plasmapy/dispersion/numerical/hirose_.py

```diff
@@ -0,0 +1,315 @@
+"""
+This module contains functionality for calculating various numerical
+solutions to Hirose's two fluid dispersion relation, see
+:cite:t:`hirose:2004` and :cite:t:`bellan:2012`.
+"""
+
+import astropy.units as u
+import numpy as np
+import warnings
+
+from astropy.constants.si import c
+from typing import Union
```

Comment:
```suggestion
from typing import Optional, Union
```

Note: `Optional[float]` is the same as `Union[float, None]`.